src/OrderFormAcceptanceTests.Steps/Steps/PassPurchasingData.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentAssertions;
+using OrderFormAcceptanceTests.Actions.Utils;
+using OrderFormAcceptanceTests.Steps.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using TechTalk.SpecFlow;
+using System.Threading.Tasks;
+
+namespace OrderFormAcceptanceTests.Steps.Steps
+{
+    [Binding]
+    internal sealed class PassPurchasingData : TestBase
+    {
+        public PassPurchasingData(UITest test, ScenarioContext context) : base(test, context)
+        {
+        }
+
+        [Then(@"a \.CSV is sent to the specified mailbox")]
```

Comment:
By suggesting to check the CSV case, does this have a knock on effect for the step description e.g. \.CSV -> \.csv? Or are you happy with it as it is? As long as the code ignores case, then works for me.

Reply:
the glue code from the feature file won't impact the assertion